src/styles/themes/default.ts

```diff
@@ -94,6 +94,7 @@ darkTheme.PAGE_BACKGROUND_COLORS = {
     [SCREENS.SETTINGS.SECURITY]: colors.ice500,
     [SCREENS.SETTINGS.STATUS]: colors.green700,
     [SCREENS.SETTINGS.ROOT]: darkTheme.sidebar,
+    [SCREENS.SETTINGS.WALLET]: '',
```

Comment:
@kubabutkiewicz Do you still need this?

Reply:
I don't need that , thanks @VickyStash fixed